package.json

```diff
@@ -19,7 +19,6 @@
     },
     "dependencies": {},
     "devDependencies": {
-        "@hubba/eslint-config-hubba": "^0.2.0",
```

Comment:
:<

Reply:
it broke the build so it is being temporarily removed, but definitely to be addressed in the near future

Reply:
is `@hubba/eslint-config-hubba` public?

Reply:
@earobinson i think the repo is but the published module isn't?